src/data/data_fetcher.py

```diff
@@ -2,7 +2,8 @@
 import os
 import json
 import time
-import urllib.request
+import requests
+import fnmatch
```

Comment:
Hadn't ever heard of this library, pretty sharp!

Reply:
I just know how to use google lol